ubuntudesign/vanilla-framework
# Rename 50 shades of greys

## Done
Renamed colour variables based on WG discussion:

```
x-dark: black
dark: #333333
mid-dark: #666666
mid-light: #cdcdcd
light: #f7f7f7
x-light: white
```

## QA
- Check build still works
- Cross ref hex codes with variable names
- Check color settings file in vf.io

## Details

Fixes #606
